tools/gnali/gnali.xml

```diff
@@ -0,0 +1,77 @@
+<tool id="gnali" name="gNALI" version="0.1.0" python_template_version="3.6">
+    <description>Get nonessential, LoF variants</description>
+    <requirements>
+        <requirement type="package" version="0.1">gnali</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        gnali -i '$test_genes' -o output
+    ]]></command>
+    <inputs>
+        <param type="data" name="test_genes" label="Test genes" format="txt" help="Specify a list of genes as HGNC symbols, separated by newline characters" />
+        <param type="select" name="database" label="Database" format="txt" help="Database to query" >
+            <option value="gnomad2.1.1">gnomAD2.1.1 (GRCh37/hg19)</option>
```

Comment:
can you set this as the default option

Reply:
cb88a86 Made gnomAD2.1.1 the default database